### PR TITLE
docs: update documentation for reading JSON from HDFS

### PR DESCRIPTION
**Pull request by bradfordb-vmware**
[skip ci]

docs for https://github.com/greenplum-db/pxf/pull/646

Authored-by: Bradford D. Boyle <bradfordb@vmware.com>

### CODE REVIEW

**Comment by lisakowen** [comment]
thanks for starting these docs, @bradfordb-vmware .  i editted a bit, and rearranged and added some info.  ready for team review.

doc review site link:  https://docs-lisa-pxf-jsontextarray.sc2-04-pcf1-apps.oc.vmware.com/pxf/6-2/using/hdfs_json.html

File: docs/content/objstore_json.html.md.erb [hunk 1]
```diff
@@ -29,7 +29,7 @@ The PXF object store connectors support reading JSON-format data. This section d
 
 Ensure that you have met the PXF Object Store [Prerequisites](access_objstore.html#objstore_prereq) before you attempt to read data from an object store.
 
-## <a id="avro_work"></a>Working with Avro Data
+## <a id="avro_work"></a>Working with JSON Data
```
Comment (ashuka24) [inline docs/content/objstore_json.html.md.erb:32]:
```suggestion
## <a id="json_work"></a>Working with JSON Data
```

File: docs/content/hdfs_json.html.md.erb [hunk 2]
```diff
@@ -266,32 +253,82 @@ CREATE EXTERNAL TABLE multiline_json_tbl(
...
+To query the JSON data in this external table:
 
 ``` sql
-CREATE EXTERNAL TABLE multiline_json_tbl(
+SELECT * FROM multiline_json_tbl;
+```
+
+## <a id="array_other"></a> Other Methods to Read a JSON Array
+
+Starting in version 6.2.0, PXF supports reading a JSON array into a `TEXT[]` column. PXF previously, and still, supports using array element projection or a single text-type column to read a JSON array. These access methods are described here.
```
Comment (ashuka24) [inline docs/content/hdfs_json.html.md.erb:272]: the previously and still statement seems a bit strange to me. Is there a way that we could rephrase this a little? Maybe something like 

`PXF still supports the old method of using array element projection [...] to read a JSON array`

**Review by mperezfuster — APPROVED** [review]
thanks for creating this!

**Comment by lisakowen** [comment]
thanks for reviewing, @ashuka24

File: docs/content/hdfs_json.html.md.erb [hunk 3]
```diff
@@ -91,8 +83,8 @@ The following table summarizes external mapping rules for JSON data.
...
 | Primitive type (integer, float, string, boolean, null) | Use the corresponding Greenplum Database built-in data type; see [Greenplum Database Data Types](https://gpdb.docs.pivotal.io/latest/ref_guide/data_types.html). |
-| Array                         | Use `TEXT`, `VARCHAR`, or `BPCHAR` to retrieve the contents of the array in one column.<br>Use <code>[<i>n</i>]</code> brackets to identify a specific array index to a member of primitive type.<br>                                                                                           |
-| Object                | Use dot `.` notation to specify each level of projection (nesting) to a member of a primitive type.                                                                                         |
+| Array                         | Use `TEXT[]` to retrieve the contents of the array in one column.                                                                                           |
```
Comment (denalex) [inline docs/content/hdfs_json.html.md.erb:86]: `contents of the array in one column` is confusing to me. With `TEXT[]`, each JSON array element will become a  GP array element. With `TEXT`, the whole JSON array will be represented as a single string containing serialized JSON array.

**Review by denalex — APPROVED** [review]

**Review by ashuka24 — APPROVED** [review]

**Pull request merged**